src/tests/test_activities.py

```diff
@@ -7,6 +7,14 @@
 class ActivitiesTestCase(BaseTestCase):
     """Test activities endpoints."""
 
+    def setUp(self):
+        """Save required roles."""
+        BaseTestCase.setUp(self)
+        self.successops_role.save()
+        self.fellow_role.save()
+        self.success_role.save()
+        self.finance_role.save()
+
     def test_create_activity(self):
         """Test that an activity has been created successfully."""
         new_activity = dict(name="tech congress",
```

Comment:
I kindly requested that you change the `activityDate` keyword arguments in this file to something dynamic. Leaving it as `2020-11-20` is unnecessary technical debt.

Reply:
Noted. Reverting accordingly.